plugins/source/aws/resources/services/lambda/functions.go

```diff
@@ -120,7 +121,9 @@ func getFunction(ctx context.Context, meta schema.ClientMeta, resource *schema.R
 
 		return err
 	}
-
+	if funcResponse.Code != nil {
+		funcResponse.Code.Location = aws.String("REDACTED_FOR_SECURITY_PURPOSES")
```

Comment:
This should be:
1. A constant defined somewhere outside of `lambda` dir, maybe in `client` pkg
2. Be documented for AWS source plugin (something like `sensitive information like source code download URLs are masked`)
3. Possibly, we want to leave the host + protocol path & update/remove only the key/presigned part

Reply:
It points to hosts that are owned and controlled by AWS so the values don't matter

Reply:
How does having the string value be a constant impact anything?

Reply:
https://go.dev/play/p/VpRRmG7Bbnl

```go
func strip(presigned string) (string, error) {
	u, err := url.Parse(presigned)
	if err != nil {
		return "", err
	}

	u.User = nil
	u.ForceQuery = false
	u.RawQuery = ""
	return u.String(), nil
}
```